promise-dialog.html

```diff
@@ -112,9 +112,9 @@
                     var target = path[i];
                     if (target.hasAttribute && (target.hasAttribute('promise-resolve') || target.hasAttribute('promise-reject'))) {
                         if (target.hasAttribute('promise-resolve')) {
-                            this.closeAsResolved(target.getAttribute("promise-value"));
+                            this.closeAsResolved(target["promise-value"]);
```

Comment:
I think this should be `this.closeAsResolved(target["promiseValue"] || target.getAttribute("promise-value"))`